float32 rot_vel #rotational velocity of the robot
